README.md

```diff
@@ -27,19 +27,19 @@ Our chat channel is on gitter [here](https://gitter.im/fossasia/open-event-atten
 ## Screenshots
 <table>
         <tr>
-<td><img src = "docs/images/screenshot_1.png" height = "480" width="270"></td>
-<td><img src = "docs/images/screenshot_2.png" height = "480" width="270"></td>
-<td><img src = "docs/images/screenshot_3.png" height = "480" width="270"></td>
+<td><img src = "fastlane/metadata/android/en-US/images/screenshot_1.png" height = "480" width="270"></td>
```

Comment:
Looking squished. Increase height

Reply:
Changed to the original size, removed ratio.